packages/components/src/dropdown-menu-v2-ariakit/index.tsx

```diff
@@ -99,6 +99,12 @@ export const DropdownMenuCheckboxItem = forwardRef<
 	);
 } );
 
+const radioCheck = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<Circle cx={ 12 } cy={ 12 } r={ 3 }></Circle>
+	</SVG>
+);
```

Comment:
Do we want to make that one an official icon in our icon library maybe? cc @jasmussen